.github/infrastructure/conformance/hashicorp/setup-hashicorp-vault-secrets.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# Notice that while hashicorp supports multiple keys in a secret,
```

Comment:
Why not add a secret with multiple keys now for use in certification tests later :)
Or you can do it in the certification test PR

Reply:
The same set of conformance test will be ran against all secret store components. Is there a way to identify during test-time if the current "store under test" supports multiple keys under the same secret? If not, we might need to consider having an __additional set of tests_ for "multiple-keys under the same secret"-stores.

Reply:
I think I get what you meant. I will test for this in the certification tests. But considering tha there are two "kinds" of secret store, it might be interesting to define a common set of tests for multi-key secret stores.